src/Tests/Unit/Adapter/OneSkyAdapterTest.php

```diff
@@ -29,8 +32,21 @@ public function setUp()
         parent::setUp();
     }
 
+
+    private function deleteFolder($path) {
+        if(!empty($path) && is_dir($path) ){
+            $dir  = new RecursiveDirectoryIterator($path, RecursiveDirectoryIterator::SKIP_DOTS); //upper dirs are not included,otherwise DISASTER HAPPENS :)
+            $files = new RecursiveIteratorIterator($dir, RecursiveIteratorIterator::CHILD_FIRST);
+            foreach ($files as $f) {if (is_file($f)) {unlink($f);} else {$empty_dirs[] = $f;} } if (!empty($empty_dirs)) {foreach ($empty_dirs as $eachDir) {rmdir($eachDir);}} rmdir($path);
```

Comment:
Maybe better to indent this block of code for better reading it?
Why you don't use Symfony Filesystem wrapper for work with filesystem?